spring-cloud-sleuth-stream/src/main/java/org/springframework/cloud/sleuth/stream/SleuthStreamProperties.java

```diff
@@ -44,4 +45,40 @@ public void setEnabled(boolean enabled) {
 	public void setGroup(String group) {
 		this.group = group;
 	}
+
+	public Poller getPoller() {
+		return this.poller;
+	}
+
+	public void setPoller(Poller poller) {
+		this.poller = poller;
+	}
+
+	public static class Poller {
```

Comment:
I'd do a quick scan of other spring projects that use properties to configure PollerMetadata, mainly to see if the naming conventions line up